src/documentation/runners/flink.md

```diff
@@ -81,6 +81,62 @@ $ mvn exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 If you have a Flink `JobManager` running on your local machine you can give `localhost:6123` for
 `flinkMaster`.
 
+Behind the hood, to create your shaded jar (containing your pipeline and the Flink runner dependencies), you have to use the `maven-shade-plugin`:
+
+```java
+                <dependency>
+                    <groupId>org.apache.beam</groupId>
+                    <artifactId>beam-runners-flink_2.10</artifactId>
+                    <version>{{ site.release_latest }}</version>
+                </dependency>
+```
+
```

Comment:
If you look at http://apache-beam-website-pull-requests.storage.googleapis.com/384/documentation/runners/flink/index.html you need some words between the code blocks because it thinks they are toggles.